platform/linuxbsd/detect.py

```diff
@@ -324,6 +325,12 @@ def configure(env):
     else:
         print("Warning: ALSA libraries not found. Disabling the ALSA audio driver.")
 
+    if env["sndio"]:
+        if os.system("pkg-config --exists sndio") == 0:  # 0 means found
+            print("Enabling sndio")
+            env.Append(CPPDEFINES=["SNDIO_ENABLED"])
+            env.ParseConfig("pkg-config --cflags --libs sndio")
```

Comment:
Since all other audio drivers and the rest of optional libs are dynamically loaded using wrappers generated by https://github.com/hpvb/dynload-wrapper, it's probably a good idea to do the same for the `sndio` (can be done in a separate PR).

Reply:
Oh, I haven't noticed that.  will study the pulseaudio example to see how it's done, thanks for pointing it out!